rules/autoconsent/complianz-opt-out.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "Complianz opt-out",
+    "prehideSelectors": [
+        "[aria-describedby=\"cookieconsent:desc\"].cc-type-opt-out"
+    ],
+    "detectCmp": [
+        {
+            "exists": "[aria-describedby=\"cookieconsent:desc\"].cc-type-opt-out"
+        }
+    ],
+    "detectPopup": [
+        {
+            "visible": "[aria-describedby=\"cookieconsent:desc\"].cc-type-opt-out"
+        }
+    ],
+    "optIn": [
+        {
+            "click": ".cc-accept-all",
+            "optional": true
+        },
+        {
+            "click": ".cc-allow",
+            "optional": true
+        },
+        {
+            "click": ".cc-dismiss",
+            "optional": true
+        }
+    ],
+    "optOut": [
+        {
+            "waitForThenClick": ".cc-deny"
```

Comment:
I've found a couple sites where this pop-up doesn't have a reject button:
- https://igdas.com.tr/
- https://www.lektion.se/
Maybe something similar to complianz-notice?

Reply:
Thanks - the `if` from complianz-notice works great here.